pages/documentation/migration-V3.md

```diff
@@ -661,16 +661,17 @@ These instructions will help you update your `@import` references to the new syn
 
    We recommend an online diff service like [quickdiff.net](https://quickdiff.net/), [diffchecker.com](https://diffchecker.com), or [text-compare.com](https://text-compare.com/).
 
-   Each of the files below is the most current version of the USWDS default settings. For each of these files, use the diff tool to find the differences between your settings file and the current defaults.
+   Each of the files below is from version 2.13.3 of the USWDS default settings. For each of these files, use the diff tool to find the differences between your settings file and the current (USWDS Version 3.X) defaults.
```

Comment:
@bonnieAcameron Curious about the parentheses surrounding `USWDS Version 3.X`. Just wanted to make sure they are intentional.

Reply:
Yes, they are!